src/tests/ftest/server/metadata.py

```diff
@@ -303,42 +303,44 @@ def test_metadata_addremove(self):
         """
         self.create_pool()
         self.container = []
+        mean_cont_cnt = 0
+        percent_cont = self.params.get("mean_percent", "/run/metadata/*")
 
         test_failed = False
         containers_created = []
         for loop in range(10):
             self.log.info("Container Create Iteration %d / 9", loop)
+            # The test will encounter ENOSPACE (-1007) while creating
+            # containers.
             if not self.create_all_containers():
                 self.log.error("Errors during create iteration %d/9", loop)
-                test_failed = True
 
             containers_created.append(len(self.container))
-
+            # We should make sure containers which are created should
+            # be destroyed without issues. Here we have to check for
+            # any container destroy errors.
             self.log.info("Container Remove Iteration %d / 9", loop)
             if not self.destroy_all_containers():
                 self.log.error("Errors during remove iteration %d/9", loop)
                 test_failed = True
+        # Calculate the mean container count
+        mean_cont_cnt = sum(containers_created) / len(containers_created)
```

Comment:
I think it would be useful to display these calculations prior to displaying the table:
```suggestion
        mean_cont_cnt = sum(containers_created) / len(containers_created)
        percent_from_mean = int(mean_cont_cnt * (percent_cont / 100))
        self.log.info(
            "Mean number of containers created in %s loops: %s",
            len(containers_created), mean_cont_cnt)
        self.log.info(
            "Max variation in number of containers from %s%% from mean: %s",
            percent_cont, percent_from_mean)
```
Feel free to fix any wording in the proposed log messages.

Reply:
Done.